src/ContentHub.php

```diff
@@ -214,6 +214,23 @@ public function deleteEntity($uuid)
     }
 
     /**
+     * Purges all entities from the Content Hub.
+     *
+     * This method should be used carefully as it deletes all the entities for
+     * the current subscription from the Content Hub.
+     */
+    public function purge() {
+        $list = $this->listEntities();
```

Comment:
I used to have a special settings configured in PHP Storm but that blew up when my laptop died. I will have to set this up again.